addons/l10n_dk/data/account_tax_template_data.xml

```diff
@@ -422,125 +600,168 @@
             }),
             (0,0, {
                 'factor_percent': -100,
                 'repartition_type': 'tax',
-                'account_id': ref('a8769'),
             }),
         ]"/>
     </record>
-
-    <record id="tax520" model="account.tax.template">
+    <record id="tax730" model="account.tax.template">
         <field name="chart_template_id" ref="dk_chart_template"/>
-        <field name="sequence">520</field>
-        <field name="name">EU Ydelseskøb</field>
+        <field name="sequence">730</field>
+        <field name="name">Naturgas- og bygasafgift</field>
         <field name="description"></field>
-        <field name="price_include" eval="0"/>
-        <field name="amount">100</field>
-        <field name="amount_type">percent</field>
+        <field name="active" eval="False"/>
+        <field name="amount">4.4490</field>
+        <field name="amount_type">fixed</field>
         <field name="type_tax_use">purchase</field>
         <field name="invoice_repartition_line_ids" eval="[(5, 0, 0),
             (0,0, {
                 'factor_percent': 100,
                 'repartition_type': 'base',
             }),
             (0,0, {
-                'factor_percent': -25,
+                'factor_percent': 100,
                 'repartition_type': 'tax',
-                'account_id': ref('a8731'),
+                'account_id': ref('a8780'),
+                'plus_report_line_ids': [ref('l10n_dk.account_tax_report_line_deduction_gas_tax')],
             }),
             (0,0, {
-                'factor_percent': 25,
+                'factor_percent': -100,
                 'repartition_type': 'tax',
-                'account_id': ref('a8740'),
+            }),
+        ]"/>
+        <field name="refund_repartition_line_ids" eval="[(5, 0, 0),
+            (0,0, {
+                'factor_percent': 100,
+                'repartition_type': 'base',
             }),
             (0,0, {
                 'factor_percent': 100,
                 'repartition_type': 'tax',
-                'account_id': ref('a8751'),
+                'account_id': ref('a8780'),
+                'minus_report_line_ids': [ref('l10n_dk.account_tax_report_line_deduction_gas_tax')],
             }),
             (0,0, {
                 'factor_percent': -100,
                 'repartition_type': 'tax',
-                'account_id': ref('a8769'),
             }),
         ]"/>
-        <field name="refund_repartition_line_ids" eval="[(5, 0, 0),
+    </record>
+    <!--  Coal tax: percentage is the same as oil and bottled gas but with it ends up in its own report line  -->
+    <record id="tax740" model="account.tax.template">
+        <field name="chart_template_id" ref="dk_chart_template"/>
+        <field name="sequence">740</field>
+        <field name="name">Kulafgift</field>
+        <field name="description"></field>
+        <field name="amount">100</field>
```

Comment:
Are you sure this needs to be a 100%?

Reply:
as discussed: yes, it's the same "Olie- og flaskegasafgift"